services/auto-idler/idle-clis.sh

```diff
@@ -31,67 +31,71 @@ echo "$ALL_ENVIRONMENTS" | jq -c '.data.developmentEnvironments[] | select((.env
   do
     PROJECT_NAME=$(echo "$project" | jq -r '.name')
     OPENSHIFT_URL=$(echo "$project" | jq -r '.openshift.console_url')
+    AUTOIDLE=$(echo "$project" | jq -r '.auto_idle')
```

Comment:
we would also need to add `auto_idle` to the graphQL request

Reply:
✔️